src/editor/Editor.js

```diff
@@ -97,8 +97,13 @@ define(function (require, exports, module) {
         USE_TAB_CHAR        = "useTabChar",
         WORD_WRAP           = "wordWrap",
         INDENT_LINE_COMMENT   = "indentLineComment";
-    
 
+   /**
+     * A list of gutter name and priorities currently registered for editors.
+     * The line number gutter is defined as { name: LINE_NUMBER_GUTTER, priority: 100 }
+     * @type {Array.<{name: string, priority: number, languages: Array}}
```

Comment:
Instead of `languages` use `languageIds`? Eventually here and through, of course